backend/app/controllers/api/v1/csrf_controller.rb

```diff
@@ -0,0 +1,9 @@
+module API
+  module V1
+    class CsrfController < BaseController
```

Comment:
I guess this controller is just for needs of tests, maybe it is even not needed 🤔